libraries/abstractions/ble_hal/test/include/iot_test_ble_hal_common.h

```diff
@@ -186,7 +186,9 @@ typedef struct
 #define bletestsMAX_PROPERTY_SIZE             30
 
 #define bletestsMTU_SIZE1                     200
-#define bletestsMAX_ADVERTISEMENT_INTERVAL    64
+
+#define bletestsMIN_ADVERTISEMENT_INTERVAL    300
```

Comment:
The range in the API header comment above is expressed in hex, but these test values are expressed in decimal. Minor inconsistency, but it made me pause.

Reply:
Fixed